### ISSUE TITLE
👍🏽 is not an emoji while 👨‍👨‍👧‍👦 is

### ISSUE THREAD

**Issue by samkelleher** [beaugunderson/emoji-aware#4]
The `.split()` can easily handle 4 part emojis like the 2 women + 2 girls (👨‍👨‍👧‍👦) - but seems to fail on a skin tone thumbs 👍🏽 up which is split into two characters (yellow thumbs up, and a skin tone square).

```
var emoji = require("emoji-aware");
console.log(emoji.isEmoji('👍🏽')); // false is logged
console.log(emoji.isEmoji('👨‍👨‍👧‍👦')); // true is logged
```

Is this a limitation in the lib?

**Comment by danbovey** [comment]
@beaugunderson I think this library would be 100% feature complete if it supported this. It's the only thing that's stopping me from finishing a fix to a chrome extension I have.

Tried looking at it but I'm really crap at regex, so I wouldn't be able to help much more than saying the range of Fitzpatrick skin tone modifiers goes from U+1F3FB  to U+1F3FF, so we need that added to the end of supported emojis <- this part probably being the hardest as you can only do this on people and body parts.

Adding skin tone emojis to the test characters gives this (it's expecting the non-modified version)
![image](https://cloud.githubusercontent.com/assets/1503076/20405464/18193ad0-ad01-11e6-9b21-f2d008085090.png)

**Comment by beaugunderson** [comment]
i'm in cambodia right now but will see if i can put together a quick fix!

---

**Comment by beaugunderson** [comment]
ok, all fixed in 3.0.1; thank you both for reporting! ✨

**Comment by danbovey** [comment]
Wow, that's a serious response time! Appreciate it!

![image](https://cloud.githubusercontent.com/assets/1503076/20414038/45372014-ad28-11e6-9881-cee832c88b49.png)
